boards/silabs/radio_boards/xg28_rb4401c/xg28_rb4401c.dts

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright (c) 2025 Shontal Biton
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ 
+/dts-v1/;
+#include <silabs/efr32zg28b322f1024im68.dtsi>
```

Comment:
The SoC support does not yet exist in this commit. Please reorder the commits to add SoC support first, then board support.